### PR TITLE
[4455] Update custom composer guide

### PR DESCRIPTION
**Pull request by bychkovdmitry**
### 🎯 Goal

Closes https://github.com/GetStream/stream-chat-android/issues/4455

Update custom composer guide.

### 🎨 UI Changes

screencapture-localhost-3000-chat-docs-sdk-android-ui-guides-custom-message-composer-2022-11-24-05_13_14

### 🧪 Testing

Launch docusaurus with `npx stream-chat-docusaurus -i -s`, and ensure that the changes look good.

#### General
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [x] PR targets the `develop` branch
- [x] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### CODE REVIEW

File: docusaurus/docs/Android/03-ui/06-guides/03-custom-message-composer.mdx [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-If the built-in [Message Composer View](../04-message-components/04-message-composer.mdx) and its available customization options don't fit your app's needs, you can create an Input View of your own.
+# Creating a Custom Message Composer
 
-Note that the UI Components Input View supports many advanced features that you'll otherwise have to implement yourself if you want to use them in your app:
+:::note
+You can find the full code from this guide on [GitHub](https://github.com/GetStream/stream-chat-android/tree/main/stream-chat-android-ui-guides/src/main/java/io/getstream/chat/android/guides/catalog/uicomponents/customcomposer). To check the final result, clone the repository and run the `stream-chat-android-ui-guides` module.
+:::
```
Comment (skydoves) [inline docusaurus/docs/Android/03-ui/06-guides/03-custom-message-composer.mdx:8]: Can we add more instructions for this, such as the image below?

Screen Shot 2022-11-24 at 11 20 59 AM

**Review by skydoves — APPROVED** [review]
Looks good to me! I left a small comment.

File: docusaurus/docs/Android/03-ui/06-guides/03-custom-message-composer.mdx (hunk 1 above)
Comment (bychkovdmitry) [reply docusaurus/docs/Android/03-ui/06-guides/03-custom-message-composer.mdx:8]: Yes, we need to improve this note everywhere. I created a task for that: https://github.com/GetStream/stream-chat-android/issues/4571

**Review by TGazica — APPROVED** [review]
Looks good to me

**Pull request merged**